OT-21 | FE - Create the Login View

auhbz opened the pull request:
Created the login page. 
All buttons (login, forgot password, create account) currently take the user to the main page.

Commit:
Wip login

need to add OFF-TOP at the top nice and big.
need to add forgot password and create account links.

Commit:
added title, create account, and forgot password

The login page is complete in terms of strtucture. For future work we will want to add/adjust functionality, and we may want to adjust the styling.

Commit:
minor clean up

Commit:
Refactor navigation

Implemented routing for the login and recording pages.
Broke up the views into their own files.

frlzjosh commented on lib/components.dart:
```diff
@@ -0,0 +1,12 @@
+import 'package:flutter/material.dart';
```
I like that you created a components file, however, I think it would be better to consider other components. So could you make components a folder and create a file inside called offTopTitle.dart

auhbz replied:
🤦‍♂ rookie mistake

frlzjosh commented on lib/main.dart:
```diff
@@ -10,149 +9,8 @@ class MyApp extends StatelessWidget {
-      home: MyHomePage(
-        title: 'Flutter Demo Home Page',
-      ),
+      title: 'title',
```
replace 'title' with 'Login Page'

frlzjosh commented on lib/websocket.dart:
```diff
@@ -1,34 +1,26 @@
-      // In case you're unable to connect to websocket try uncommenting this string below
-      // "ws://10.0.2.2:8080/name" 
-    );
+    final channel = IOWebSocketChannel.connect("ws://localhost:9000/name"
```
thank you!

frlzjosh reviewed (changes requested):
This branch was well done. I do have a few changes requested. 

I also think that it only makes sense to create a header on the loginPage above the form that says "Off-Top Login"

Commit:
PR Changes requested

- added a components folder for offTopTitle
- changed ‘title’ in main to ‘Login Page’
- added a header for the login page

frlzjosh committed:
including the ability to access the websocket again

frlzjosh reviewed (approved):
nice!
